WordPress/Classes/ViewRelated/Site Creation/WebAddress/WebAddressWizardContent.swift

```diff
@@ -304,11 +307,23 @@ final class WebAddressWizardContent: CollapsableHeaderViewController {
 
         view.addSubview(noResultsLabel)
 
+        let noResultsLabelTopAnchor = noResultsLabel.topAnchor.constraint(equalTo: searchHeader.bottomAnchor)
+        noResultsLabelTopAnchor.identifier = WebAddressWizardContent.noResultsLabelTopAnchorConstraintIdentifier
+
         NSLayoutConstraint.activate([
-            noResultsLabel.widthAnchor.constraint(equalTo: headerView.widthAnchor),
+            noResultsLabel.widthAnchor.constraint(equalTo: table.widthAnchor, constant: -50),
             noResultsLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            noResultsLabel.topAnchor.constraint(equalTo: headerView.bottomAnchor, constant: Metrics.noResultsTopInset)
+            noResultsLabelTopAnchor
         ])
+
+        updateNoResultsLabelTopInset()
+    }
+
+    /// Sets the top inset for the noResultsLabel based on layout orientation
+    private func updateNoResultsLabelTopInset() {
+        if let constraint = view.constraints.first(where: { $0.identifier == WebAddressWizardContent.noResultsLabelTopAnchorConstraintIdentifier }) {
```

Comment:
Not a blocker at all, but we could also store the constraint itself as a private optional property so we wouldn't have to search for it using the identifier.

`private var noResultsLabelTopAnchor: NSLayoutConstraint?` (optional so we don't have to worry about init)

We'd set the property after its creation on line #310, and this would turn `updateNoResultsLabelTopInset()` into:

`noResultsLabelTopAnchor?.constant = UIDevice.current.orientation.isPortrait ? Metrics.noResultsTopInset : 0`

Reply:
Updated according to this suggestion in [0d31bde](https://github.com/wordpress-mobile/WordPress-iOS/pull/18123/commits/0d31bde0d487035572dcd7059d744311aa0b7a49) 🙇